src/common/utils/getAccountCodes.js

```diff
@@ -0,0 +1,9 @@
+import { get, uniq } from 'lodash';
+
+// eslint-disable-next-line import/prefer-default-export
+export const getAccountCodes = (vendor) => {
```

Comment:
is it common util?

Reply:
I think so. I'm going to use it for `InvoiceLineForm` too.